libbeat/common/kafka/version.go

```diff
@@ -57,7 +57,16 @@ var (
 		"2.4": sarama.V2_4_0_0,
 		"2.5": sarama.V2_5_0_0,
 		"2.6": sarama.V2_6_0_0,
+		"2.7": sarama.V2_7_0_0,
```

Comment:
supported versions were referenced from here https://docs.aws.amazon.com/msk/latest/developerguide/supported-kafka-versions.html

Reply:
Maybe we should add a comment referencing this link if we plan on following the same supported versions. When the time comes to revisit this list one can quickly get this list up to date when necessary. Could be right above `truncatedKafkaVersions` definition.